apps/web/domains/landing-page/Hero.tsx

```diff
@@ -106,6 +107,9 @@ export const Hero = () => {
                     <li className="mr-9 font-medium hover:text-gray-700">
                       <a href="#perguntas-frequentes">Perguntas Frequentes</a>
                     </li>
+                    <li className="mr-9 font-medium hover:text-gray-700">
+                      <ToggleTheme />
```

Comment:
Yes, they can be customized inside the component itsef. I've just used these "🌞" and "🌜" because they work in every client, no doubt, and I didn't want to load any other assets into client. You can put an SVG too, instead. And even better, do some animation using the View Transition API.

Reply:
Would you be able to update the icons for to the ones used at https://ui.shadcn.com/?

Reply:
> Would you be able to update the icons for to the ones used at https://ui.shadcn.com/?

for sure! :D

Reply:
Thank you @samuelbelo 

I'm a little concerned about this one because the dark mode is not that great yet (specially for inputs).

And I'm also concerned that adding a dark mode will force us to support dark design for each new component.

Taking into consideration that we don't have much time to work on this project, I fear that the advantages of having a dark mode won't be enough for the overhead in effort that it will bring to us.

@anti-duhring what do you think?

Reply:
Although creating a dark mode may increase the amount of design work, we can mitigate this through strategic implementation. For instance, we could adopt a design system that uses CSS variables for colors. This would allow us to easily switch colors for dark mode, without having to create a separate design for each component.

Also, if time is a concern, we could consider implementing dark mode as an incremental improvement. We don't have to do it all at once. We can start with the most visible and used parts of the application and gradually add dark mode support to other parts over time.

Anyway, I'm glad to contribute with any ideas, whatsoever :)

Reply:
I mean, are you capable of being in charge of the dark-mode feature if we implement it?

As of me, I think that are other higher-priorities tasks on the roadmap of this project (user preferences, add an opening, supporting companies page, jobs board) other than the dark mode.

Which means I wouldn't want to deal with it as it would slow me down.

But if you ensure that you're capable of being responsible to handle the dark-mode alternatives and fix the issues that arrises with it, I'm fine with approving this PR.

The thing is that this is a side-project and we all have 9-5 jobs so we need to choose wisely how we want to spend our time.

If you're capable of, then solve the merge conflicts and I'll aprove it.

Otherwise, I'm sorry but we'll have to reject it.

Reply:
I understand, no problem. I can contribute with more urgent tasks, I'm here to help! 😃